raw_topic: "/bytes"

port_name: "/dev/ydlidar"
baudrate: 128000

frequency_read: 5000

scan_topic: "/scan_frame" 

max_angle: 360 
min_angle: 0

max_distance: 10
min_distance: 0.13

frame_id: "lase_frame"
